package.json

```diff
@@ -5,7 +5,8 @@
   "scripts": {
     "start": "vue-cli-service serve",
     "build": "vue-cli-service build",
-    "lint": "vue-cli-service lint"
+    "lint": "vue-cli-service lint",
+    "deploy": "npm run build; gh-pages -add -d dist"
```

Comment:
s/npm/yarn/

Reply:
It should probably be `&&` instead of `;` to stop the deployment in case there's a build error.